runtime/jvmti/jvmtiGeneral.c

```diff
@@ -65,9 +65,9 @@ static const J9JvmtiErrorMapping errorMap[] = {
 	{ "JVMTI_ERROR_NAMES_DONT_MATCH" , 69 },
 	{ "JVMTI_ERROR_UNSUPPORTED_REDEFINITION_CLASS_MODIFIERS_CHANGED" , 70 },
 	{ "JVMTI_ERROR_UNSUPPORTED_REDEFINITION_METHOD_MODIFIERS_CHANGED" , 71 },
-#if defined(J9VM_OPT_VALHALLA_NESTMATES)
+#if defined(J9VM_OPT_VALHALLA_NESTMATES) || (JAVA_SPEC_VERSION >= 15)
```

Comment:
nestmates was added in java 11.  This can be #ifdef'd on that version
```suggestion
#if (JAVA_SPEC_VERSION >= 11)
```